### PR TITLE
Fix verify.rs

### PR DESCRIPTION
**Pull request by dns2utf8**
Finishing d2f8fb0 from @jseyfried

### CODE REVIEW

File: src/libsyntax/symbol.rs [hunk 1]
```diff
@@ -35,6 +35,10 @@ impl Symbol {
         with_interner(|interner| interner.gensym(string))
     }
 
+    pub fn invalid() -> Self {
+        Symbol(0u32)
+    }
+
```
Comment (dns2utf8) [inline src/libsyntax/symbol.rs]: I had to create this method because `Symbol.0` is private. Is this ok?

Comment (petrochenkov) [inline src/libsyntax/symbol.rs]: `Symbol(0)` is already available as `keywords::Invalid`

Comment (petrochenkov) [inline src/libsyntax/symbol.rs]: `keywords::Invalid.name()` to be precise

Comment (dns2utf8) [inline src/libsyntax/symbol.rs]: ah, thank you 😄

**Pull request merged**